pom.xml

```diff
@@ -263,12 +304,12 @@
             <version>${codice-test.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
+        <!--<dependency>
```

Comment:
Should I delete this block?

Reply:
Sounds good to me assuming everything else is passing.